editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -2119,9 +2120,12 @@ void SpatialEditorViewport::_notification(int p_what) {
 
 		bool visible = is_visible_in_tree();
 
+
```

Comment:
This will add a style issue, as `clang-format` ensures that we don't have more than 1 consecutive empty lines. See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html